modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/util/PowwowUtil.java

```diff
@@ -436,4 +533,48 @@ private static void _sendNotificationEvent(
 			powwowParticipant.getParticipantUserId(), notificationEvent);
 	}
 
+	private static final String STR_SPACE = " ";
+
+	private static final Map<Frequency, String> _frequencyMapLabel = new HashMap<>();
+
+	private static final Map<Integer, String> _monthMapLabel = new HashMap<>();
+
+	private static final Map<Integer, String> _positionMapLabel = new HashMap<>();
+
+	private static final Map<Weekday, String> _weekDayMapLabel = new HashMap<>();
+
+	private static final ResourceBundle resourceBundle = ResourceBundle.getBundle("content.Language");
+
+	static {
+		_frequencyMapLabel.put(Frequency.DAILY, LanguageUtil.get(resourceBundle, "days"));
```

Comment:
The purpose of `LanguageUtil.get` is provide the flexibility to translate by language. So, putting it in static constants does not make sense.